server/package/extractor/doc-generator.tsx

```diff
@@ -338,6 +472,11 @@ async function convertAndWriteDocs(
     throw new Error("Compile error");
   }
 
+  await updateReadmeRelativeLinks(
```

Comment:
We'd also need to do this as a post-load script. 

The `updateReadmeRelativeLinks` would only work for newly generated docs.

Reply:
Yes. Currently it only works for newly generated docs. But, yes we can do that as a post load script.

Reply:
@pastelsky what do you mean by a post-load script?

Reply:
@pastelsky need some discussion on this